assets/schema_input.json

```diff
@@ -10,9 +10,14 @@
             "sample": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "meta": ["id"],
+                "meta": ["irida_id"],
                 "errorMessage": "Sample name must be provided and cannot contain spaces"
             },
+            "sample_name": {
+                "type": "string",
+                "meta": ["id"],
+                "errorMessage": "Sample name must be provided"
```

Comment:
Are there no restrictions that need to be mentioned in the error message? Also, I don't know if the same name needs to be provided, since it's not required.

Reply:
You're right! I actually suggested the same change in the other PR :man_facepalming:

Changing it to "Optional. Used to override sample when used in tools like IRIDA-Next"

Reply:
Thank you for the reminder Steven! I have to make this change in `speciesabundance` 😅